tests/lib/location/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(location_test)
+
+# Generate runner for the test
+test_runner_generate(src/location_test.c)
+
+cmock_handle(../../../include/modem/modem_key_mgmt.h)
+cmock_handle(../../../include/modem/modem_info.h)
```

Comment:
Is the modem_info mock needed, I don't see it being used?

Reply:
True, doesn't seem to be needed anymore.